website/docs/plugin/framework/handling-data/types/map.mdx

```diff
@@ -0,0 +1,127 @@
+* [`types.MapUnknown(attr.Type) types.Map`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapUnknown): An unknown list value with the given element type.
+* [`types.MapValue(attr.Type, map[string]attr.Value) (types.Map, diag.Diagnostics)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValue): A known value with the given element type and values.
+* [`types.MapValueFrom(context.Context, attr.Type, any) (types.Map, diag.Diagnostics)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValueFrom): A known value with the given element type and values. This can convert the source data from standard Go types into framework types as noted in the documentation for each element type, such as giving `map[string]*string` for a `types.Map` of `types.String`.
+* [`types.MapValueMust(map[string]attr.Type, map[string]attr.Value) types.Map`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValueMust): A known value with the given element type and values. Any diagnostics are converted to a runtime panic. This is recommended only for testing or exhaustively tested logic.
+
+In this example, a known map value is created from framework types:
+
+```go
+elements := map[string]attr.Value{
+    "key1": types.StringValue("value1"),
+    "key2": types.StringValue("value2"),
+}
+listValue, diags := types.MapValue(types.StringType, elements)
+```
+
+Otherwise, for certain framework functionality that does not require `types` implementations directly, such as:
+
+* [`(tfsdk.State).SetAttribute()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/tfsdk#State.SetAttribute)
+* [`types.ListValueFrom()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#ListValueFrom)
+* [`types.MapValueFrom()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValueFrom)
+* [`types.ObjectValueFrom()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#ObjectValueFrom)
+* [`types.SetValueFrom()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#SetValueFrom)
+
+A Go built-in map of string key type (`map[string]T`) or type alias of a map of string key type such as `type MyMapType map[string]T` can be used instead.
+
+In this example, a `map[string]string` is directly used to set a map attribute value:
+
+```go
+elements := map[string]string{
+    "key1": "value1",
+    "key2": "value2",
+}
+diags := resp.State.SetAttribute(ctx, path.Root("example_attribute"), elements)
+```
+
+In this example, a `types.List` of `types.String` is created from a `[]string`:
+
+```go
+elements := map[string]string{
+    "key1": "value1",
+    "key2": "value2",
+}
+listValue, diags := types.MapValueFrom(ctx, types.StringType, elements)
```

Comment:
Nit: Could use `mapValue` to emphasise type in var.

Reply:
Good copypasta 🍝  catch, will update!